### PR TITLE
fix: detect screen reader keyboard events

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Related to #1595

### CODE REVIEW

**Review by dauriamarco — APPROVED** [review]
LGTM 👍

File: src/global/helpers/eventing/fake-event-detection.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/** Gets whether an event could be a faked `mousedown` event dispatched by a screen reader. */
```
Comment (kyubisation) [inline src/global/helpers/eventing/fake-event-detection.ts:1]: Add an attribution to angular/components

File: src/global/helpers/eventing/input-modality-detector.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
+import {
```
Comment (kyubisation) [inline src/global/helpers/eventing/input-modality-detector.ts:1]: Add an attribution to angular/components

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**